File: 2114.cpp
class Solution 
{
public:
    int mostWordsFound(vector<string>& sentences) 
    {
        int count=INT_MIN;
        for(auto x:sentences)
        {
            int c=0;
            for(int i=0;i<x.length();i++)
            {
                if(x[i]==' ')
                {
                    c++;
                }
            }
            count=max(count,c);
        }
        return count+1;
    }
};

File: 2114.py
#Method-1 : 

class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        res=0;
        for word in sentences:
            res=max(res,len(word.split()))
        return res
      
 #Method-2 : 

class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        return max(len(words.split()) for words in sentences)
